rest/model/distro.go

```diff
@@ -133,3 +133,37 @@ func (e *APIExpansion) ToService() (interface{}, error) {
 
 	return interface{}(d), nil
 }
+
+// APIDistroSetup is derived from a service layer set-up script (string)
```

Comment:
I think you can just use the API distro struct as it has a Setup field with the right type and tag...